easybuild/tools/options.py

```diff
@@ -823,12 +829,18 @@ def _postprocess_config(self):
         if self.options.pretend:
             self.options.installpath = get_pretend_installpath()
 
+
         if self.options.robot is not None:
             # paths specified to --robot have preference over --robot-paths
             # keep both values in sync if robot is enabled, which implies enabling dependency resolver
             self.options.robot_paths = [os.path.abspath(path) for path in self.options.robot + self.options.robot_paths]
             self.options.robot = self.options.robot_paths
 
+        # Update the search_paths keeping it in sync with robot_paths
+        self.options.search_paths = [os.path.abspath(path) for path in
+                                     self.options.robot_paths + self.options.search_paths]
```

Comment:
I would drop the `robot_paths` here, and just do `abspath` on the entries in `search_paths` (if any)